### ISSUE TITLE
Thumbnail in Recent Apps shows settings menu

### ISSUE THREAD

**Issue by ioana-chiorean** [mozilla-mobile/focus-android#924]
Nexus 7 - Android 5.1.2
Pixel - Android 7.1.2
Focus 1.1. Beta Build #2202

Steps:
- do some browsing and play with stealth setting on/on 
- press recent tabs in android
OR:
- open a link from another app while focus is default browsers
- press recent tabs in android

Expected results:
- the thumbnail should be blank

Actual results:
- after 2,3 pages and one on/off changes the thumbnail shows the settings menu 

See video:
- https://youtu.be/JKzgqXZ1odM
- https://youtu.be/ejaBLD2BPTo

![photo_2017-07-14_14-47-22](https://user-images.githubusercontent.com/1391451/28211313-a0b6dc1c-68a4-11e7-9d00-5ce69db7abaf.jpg)
![photo_2017-07-14_14-47-21](https://user-images.githubusercontent.com/1391451/28211314-a0b9768e-68a4-11e7-9622-62449ea294dc.jpg)

**Comment by ioana-chiorean** [comment]
@pocmo the only problem I see is that I was not in settings menu. So in this case for me is not logical to show it.

**Comment by pocmo** [comment]
Yeah, I agree that is confusing at times. Android always shows the screenshot of the last time it was allowed to (and actually did) take a screenshot. This doesn't make much sense - but we can't change the behavior unfortunately. :(

**Comment by ioana-chiorean** [comment]
I was not aware of that - thanks for the clarification.
